toy_2d/src/ilqr.py

```diff
@@ -258,12 +258,13 @@ def animate(self, init_state, controls, show_vis=False):
         states = self.lcs.system.state_history
         controls = self.lcs.system.control_history
         control_forces, control_locs = controls[:, :2], controls[:, 2:]
+        gif_name = "gif_" + str(self.test_num)
```

Comment:
Could the `gif_name` instead be something like `"ilqr_result_" + str(self.test_num)`?  Just so the name is more expressive.  Also the file extension is `.gif` anyway so the title starting with `gif_` isn't informative.